### PR TITLE
Update secp256k1

### PR DESCRIPTION
**Pull request by subtly**

### CODE REVIEW

**Comment by subtly** [comment]
Waiting on arkpar's PR which fixes tests.

File: secp256k1/libsecp256k1-config.h [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+#define PACKAGE_URL ""
+
+/* Define to the version of this package. */
+#define PACKAGE_VERSION "0.1"
+
+/* Define to 1 if you have the ANSI C header files. */
+#define STDC_HEADERS 1
+
+/* Define this symbol to enable x86_64 assembly optimizations */
+/* #undef USE_ASM_X86_64 */
```
Comment (chriseth) [inline secp256k1/libsecp256k1-config.h]: We could activate that later.

**Comment by gavofyork** [comment]
still have windows build fail.

**Comment by CJentzsch** [comment]
this fixes the failing consensus tests (https://github.com/ethereum/tests/pull/94/files)

**Pull request merged**